equinor/EchoComponents
# chore: #79250 - Upgrade EDS to 0.21.0

## Prerequisites

Please make sure you can check the following boxes:


## Type(s) of Changes

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:


Description
Updating to EDS v0.21.0 (from 0.19.0). This includes a breaking change with `SingleSelect` and `MultiSelect` which are both deprecated and replaced by `Autocomplete` component. 

I updated the `TimePicker` Echo Component to be using `Autocomplete`:
As Autocomplete uses Downshift, it was updated with some extra padding on the input, which I had to fix for the placeholder to be displayed correctly. 

v0.21.0 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.21.0

v0.20.4 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.20.4

v0.20.3 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.20.3

v0.20.2 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.20.2

v0.20.1 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.20.1

v0.20.0 release notes:
https://github.com/equinor/design-system/releases/tag/eds-core-react%400.20.0

### How to test:
- npm run storybook
- check every component: see if you experience anything weird
  - focus on TimePicker - this is affected by the breaking changes (SingleSelect deprecated > Replaced by Autocomplete)
  
## Related PRs:
EchoFramework: https://github.com/equinor/EchoFramework/pull/297
